java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -0,0 +1,456 @@
+     * @param key The <code>key</code> of the JSON document.
+     * @return Returns a string representation of the JSON document. If <code>key</code> doesn't
+     *     exist, returns null.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.get(client, "doc").get();
+     * assert value.equals("{'a': 1.0, 'b': 2}");
+     * }</pre>
+     */
+    public static CompletableFuture<String> get(@NonNull BaseClient client, @NonNull String key) {
+        return executeCommand(client, new String[] {JSON_GET, key});
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @return Returns a string representation of the JSON document. If <code>key</code> doesn't
+     *     exist, returns null.
+     * @example
+     *     <pre>{@code
+     * GlideString value = client.Json.get(client, gs("doc")).get();
+     * assert value.equals(gs("{'a': 1.0, 'b': 2}"));
+     * }</pre>
+     */
+    public static CompletableFuture<GlideString> get(
+            @NonNull BaseClient client, @NonNull GlideString key) {
+        return executeCommand(client, new GlideString[] {gs(JSON_GET), key});
+    }
+
+    /**
+     * Retrieves the JSON value at the specified <code>paths</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param paths List of paths within the JSON document.
+     * @return
+     *     <ul>
+     *       <li>If one path is given:
+     *           <ul>
+     *             <li>For JSONPath (path starts with <code>$</code>): Returns a stringified JSON list
+     *                 replies for every possible path, or a string representation of an empty array,
+     *                 if path doesn't exist. If <code>key</code> doesn't exist, returns None.
+     *             <li>For legacy path (path doesn't start with <code>$</code>): Returns a string
+     *                 representation of the value in <code>paths</code>. If <code>paths</code>
+     *                 doesn't exist, an error is raised. If <code>key</code> doesn't exist, returns
+     *                 None.
+     *           </ul>
+     *       <li>If multiple paths are given: Returns a stringified JSON, in which each path is a key,
+     *           and it's corresponding value, is the value as if the path was executed in the command
+     *           as a single path.
+     *     </ul>
+     *     In case of multiple paths, and <code>paths</code> are a mix of both JSONPath and legacy
+     *     path, the command behaves as if all are JSONPath paths.
+     * @example
+     *     <pre>{@code
+     * String value = client.Json.get(client, "doc", new String[] {"$.a", "$.b"}).get();
```

Comment:
I guess you need 2 examples here - with 1 and >1 paths. One of the paths could start with `$`, another isn't - for variety.